content/en/_index.md

```diff
@@ -872,6 +1042,11 @@ try {
     // IllegalArgumentException is thrown when an invalid argument has been passed to a function.
 }
 {{< /tab >}}
+{{< tab header="Terraform" >}}
+Because they increase cyclomatic complexity, do not use error handling such as
```

Comment:
Prioritizing between guidelines is something we avoid, instead prefer to cite the principles: are we saying this error handling technique is not in line with our simplicity goals?

We also prefer positive examples of what to do, rather than focusing on anti-patterns: what is the error handling we want samples to use? I expect it is mostly "none" because we want to rely on terraform to fall through to default logging, both for our maintenance and for developer practice.

suggestion: Start by recommending the error handling practice we want, then add special considerations around try.

#StyleGuideFeedback #TerraformPracticeFeedback

Reply:
done